src/main/java/internetShop/controller/RegistrationController.java

```diff
@@ -0,0 +1,38 @@
+package internetShop.controller;
+
+import internetShop.entity.User;
+import internetShop.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class RegistrationController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/registration")
+    public String registration(Model model) {
+        model.addAttribute("userForm", new User());
+        return "/registration";
```

Comment:
/registration extract to constant

Reply:
Something like this?
 private static final String registration = "/registration";
 @GetMapping(registration)..